server/routes/producers-router.js

```diff
@@ -13,7 +13,24 @@ module.exports = function producersRouter(producers) {
       .catch(err => next(err))
   })
 
-  router.post('/', (req, res, next) => {
+  router.put('/edit-profile/', (req, res, next) => {
```

Comment:
Instead of `PUT /edit-profile` and `POST /create-profile`, you can use the same path for both. The create/update is implied in the HTTP method.

`router.put('/', handerFunctionForUpdate)`
`router.post('/', handerFunctionForCreate)`